### DISCUSSION TITLE
postgresql catalog is not displaying in trino

### DISCUSSION THREAD

**Discussion by Mathuraju** [trinodb/trino#14854] [General]
I have created a "postgres.properties" catalog under \etc\catalog\ folder and restarted the trino server. I have run show catalogs command but it is not displaying postgres catalog.

trino>
D:\Learnings\Presto\codebase\trinodb_trino>docker exec -it trino trino
trino> show catalogs;
 Catalog
---------
 jmx
 memory
 system
 tpcds
 tpch
(5 rows)

Query 20221101_134701_00000_igacn, FINISHED, 1 node
Splits: 7 total, 7 done (100.00%)
0.61 [0 rows, 0B] [0 rows/s, 0B/s]

**Comment by nineinchnick** [comment]
PostgreSQL is not configured by default, you have to add a catalog properties file for it. See https://trino.io/docs/current/installation/containers.html#configuring-trino

**Reply by Mathuraju** [reply]
@nineinchnick : YEs, I have added a catalog file under etc\catalog folder with the following properties but still did not appear in the show caalog command.
file name: postgres.properties
properties:
connector.name=postgresql
connection-user=postgres
connection-password=HznXqHRL7a
connection-url=jdbc:postgresql://postgres:5432/postgres
case-insensitive-name-matching= true

**Reply by nineinchnick** [reply]
Please read the docs I linked. It mentions how to mount additional files as a volume in the container.
